README.md

```diff
@@ -10,38 +10,27 @@ The original README.md is moved to [`tests/README.md`](tests/README.md).
 git clone git@github.com:hep-lbdl/hadml.git
 cd hadml
 
-# [OPTIONAL] create conda environment
-conda create -n herwig python=3.9
```

Comment:
Maybe we should still have this command, so that it's obvious which python version should have all the correct versions of dependencies available?